### PR TITLE
[bitnami/wordpress] Add flag -r to xargs to avoid failure on empty input

### LINKED ISSUE #9074: Init-container wordpress terminates on chown
### Name and Version

bitnami/wordpress-13.0.11

### What steps will reproduce the bug?

In OVH.com public cloud Kubernetes I 

created a pvc like this:

```yaml
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: michiel-wp-pvc
spec:
  accessModes:
    - ReadWriteMany
  resources:
    requests:
      storage: 1Gi
  storageClassName: csi-cinder-classic
```
And then run the install with this:
```
 helm install wp-michiel5 bitnami/wordpress -f values.yaml 
```
with below values.yaml

### Are you using any custom parameters or values?

```yaml
persistence:
  existingClaim: michiel-wp-pvc
  
volumePermissions:
  enabled: true

ingress:
  enabled: false

service:
  type: NodePort

mariadb:
  enabled: false
  
wordpressScheme: https

externalDatabase:
  host: xx
  user: xx
  database: xx
  password: xx
  port: 3306
```

### What is the expected behavior?

The wordpress install would complete and a working wordpress pod becomes available.

### What do you see instead?

In the pod details, I see that the init-container has exited on `chown` with exit code 123. 

```yaml
Init Containers:
  volume-permissions:
    Container ID:  containerd://5832981fc897b4040e19ae4ae2a7859fcffb52334790fc5b04f843baa7b76001
    Image:         docker.io/bitnami/bitnami-shell:10-debian-10-r337
    Image ID:      docker.io/bitnami/bitnami-shell@sha256:3b73ef5913dc5a23a4dd2c60d004f656a7cacd257d5e2bd4a0a0c218d242439d
    Port:          <none>
    Host Port:     <none>
    Command:
      /bin/bash
    Args:
      -ec
      mkdir -p /bitnami/wordpress
      find /bitnami/wordpress -mindepth 1 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | xargs chown -R 1001:1001
      
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error
      Exit Code:    123
      Started:      Fri, 18 Feb 2022 13:52:11 +0100
      Finished:     Fri, 18 Feb 2022 13:52:11 +0100
    Ready:          False
    Restart Count:  5
    Environment:    <none>
    Mounts:
      /bitnami/wordpress from wordpress-data (rw,path="wordpress")
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-2cf9l (ro)
```

### Additional information

When I start a bash container based on the  docker.io/bitnami/bitnami-shell:10-debian-10-r337 with the same volume mount, it strikes me that the volume-mount is completely empty (i'm not shure if that is expected at this point). 

Now I can replicate the 123 exit code by:
```
root@my-debug-helm:/# find /bitnami/wordpress  -mindepth 1 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | xargs chown -R 1001:1001
chown: missing operand after '1001:1001'
Try 'chown --help' for more information.
root@my-debug-helm:/# echo $?
123
```
My analysis is that the find-xargs-chown sequence does not work on an empty dir! 
The error disappears after I put something there.
```
echo hello > /bitnami/wordpress/readme.txt
```
Now, repeating the find-xargs-chown there is no error. Also now I can do the helm install again and it doesn't crash on that thing anymore !

### PR DESCRIPTION
**Pull request by migruiz4**
Signed-off-by: Miguel Ruiz <miruiz@vmware.com>

**Description of the change**
Replaces the `find | xargs` logic with `chown -R`, as `xargs` would fail with error `missing operand after` if `find` didn't find any coincidence`.

There are other alternatives that are discussed here: https://unix.stackexchange.com/questions/491948/chown-missing-operand-after-rootusers

Such as using `find -exec` or adding `xargs` flag `-r` to not fail if no args are provided, but I decided to keep it simple and go for the `chown -R` as we do in other charts.

**Applicable issues**

  - fixes #9074

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the values.yaml and added to the README.md using (readme-generator-for-helm)[https://github.com/bitnami-labs/readme-generator-for-helm]
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/wordpress/templates/deployment.yaml [hunk 1]
```diff
@@ -84,9 +84,9 @@ spec:
             - |
               mkdir -p /bitnami/wordpress
               {{- if eq ( toString ( .Values.volumePermissions.containerSecurityContext.runAsUser )) "auto" }}
-              find /bitnami/wordpress -mindepth 1 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | xargs chown -R $(id -u):$(id -G | cut -d " " -f2)
+              chown -R $(id -u):$(id -G | cut -d " " -f2) /bitnami/wordpress
```
Comment (juan131) [inline bitnami/wordpress/templates/deployment.yaml:87]: We should definitely keep excluding some files that are common in mounted volumes. I mean, keeping this part: `-not -name ".snapshot" -not -name "lost+found`.

**Review by juan131 — CHANGES_REQUESTED** [review]
Please find a way to avoid "xargs" to execute the "chown" when no files are found.

**Comment by migruiz4** [comment]
After some local testing, I think the best option would be to add the flag `-r`:

```
$ man xargs
       -r, --no-run-if-empty
              If  the standard input does not contain any nonblanks, do not run the command.  Normally, the command is run once even if there is no input.  This option is a GNU extension.
```

Local tests:
```console
$ find ./ \( ! -user bitnami -o ! -group bitnami \) -print0 | xargs chown bitnami:bitnami
chown: missing operand after ‘bitnami:bitnami’
Try 'chown --help' for more information.
$ find ./ \( ! -user bitnami -o ! -group bitnami \) -print0 | xargs -r chown bitnami:bitnami
```

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**